android/src/main/java/com/swmansion/reanimated/nodes/PropsNode.java

```diff
@@ -87,6 +87,11 @@ protected Double evaluate() {
               throw new IllegalArgumentException("Unexpected type " + type);
           }
         }
+      } else if (node instanceof ConcatNode) {
+        String key = entry.getKey();
+        String value = (String) node.value();
+        mPropMap.putString(key, value);
+        hasUIProps = true;
       } else {
         String key = entry.getKey();
         if (mNodesManager.uiProps.contains(key)) {
```

Comment:
Maybe a small suggestion:
Does it make sense use a type check in the else block instead of a new else if block specially for ConcatNode.
In the future the might be more Nodes that do string operators.

I'm thinking about doing it the same it is done in line#75

Reply:
Yeah most probably, I just made a quick patch to get it working. I'm thinking @kmagiera and @osdnk will probably have better insight into the rest of the codebase than me, perhaps they can take over from here? Otherwise, once they review it I can probably make any needed changes.